CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [4.0.2] - 2019-08-27
```

Comment:
I'd say this is a minor revision: Backward compatible but functionally different because you can use IDs were before you couldn't. That would make it `4.1.0`. Checkout https://semver.org for a more detailed explanation for the way we version things at Brandwatch,